sdk/core/core-http/lib/util/xml.browser.ts

```diff
@@ -147,3 +139,40 @@ function buildNode(obj: any, elementName: string): Node[] {
     throw new Error(`Illegal value passed to buildObject: ${obj}`);
   }
 }
+
+function buildAttributes(doc: any, attrs: { [key: string]: { toString(): string } }): Attr[] {
```

Comment:
Please move this function back to its original position i.e. before the function `buildNode` so that the diff view can show exactly what changed in this PR when it comes to `buildAttributes`